### PR TITLE
Heatmap example

### PR DESCRIPTION
**Pull request by vcasadei**

### CODE REVIEW

**Comment by vcasadei** [comment]
@ramonza could you review this please?

**Comment by ramonza** [comment]
The heat map looks way too regular to be realistic - the gradient is pixel perfect. How could this possibly represent a real latency, even one locally you would expect some jitter? 

Is there a way to change the Y-axis labels to not print out the whole name of the metric? Ideally it would only be labeled with the "le" tag value. 

Another issue (not sure if it's possible to fix) is that the +Inf is at the bottom where logically it should be at the top (above the highest value).

**Comment by vcasadei** [comment]
@ramonza I did some tests and I can display the value of the le= tag without the le, this way, I will have the values in the y-axis and with this configuration, the +Inf stays on top.

File: README.md [hunk 1]
```diff
@@ -93,6 +93,32 @@ The generated path will look like:
 #### Graph visualization on Graphite
 ![Graph visualization on Graphite](https://i.imgur.com/A4AExV8.png)
 
+#### Heatmap visualization on Grafana
+
+On Grafana it's possible to generate heatmaps based on time series bucket. To do so, it's necessary to configurate the Axes, setting the `Data format` to `Time series bucket` as shown in the image below:
```
Comment (rakyll) [inline README.md]: configure

File: README.md [hunk 2]
```diff
@@ -93,6 +93,32 @@ The generated path will look like:
...
+
+With this configuration, Grafana automatically identifies the bucket boundaries in the data that's being sent and generate the correct heatmap without the need of further configuration.
+
+In the next image, it's possible to see the heatmap created from  a gRPC client latency view, that can be found in the ocgrpc package as ClientRoundtripLatencyView.
+
+The code for generating this example is not much different from the grpc example contained in the example folder. The main change is on line 45 of the client:
+
+```go
+...
+    // Register the view to collect gRPC client stats.
```
Comment (rakyll) [inline README.md]: Remove the trailing whitespace here.

**Comment by vcasadei** [comment]
Hi @rakyll I committed the README changes.

File: README.md [hunk 3]
```diff
@@ -93,6 +93,32 @@ The generated path will look like:
...
+
+With this configuration, Grafana automatically identifies the bucket boundaries in the data that's being sent and generate the correct heatmap without the need of further configuration.
+
+In the next image, it's possible to see the heatmap created from  a gRPC client latency view, that can be found in the ocgrpc package as ClientRoundtripLatencyView.
+
+The code for generating this example is not much different from the grpc example contained in the example folder. The main change is on line 45 of the client:
+
+```go
+...
+// Register the view to collect gRPC client stats.
```
Comment (rakyll) [inline README.md]: What I meant was to remove the tabs entirely

```
// Register the view to collect gRPC client stats.
if err := view.Register(ocgrpc.ClientRoundtripLatencyView); err != nil {
	log.Fatal(err)
}
```

**Comment by rakyll** [comment]
Done, thanks.

**Pull request merged**